src/Pages/LoginPage/LoginPage.js

```diff
@@ -0,0 +1,51 @@
+import { html, css, LitElement } from 'lit-element';
+import defaultStyles from '../../FeApp.style.js';
+import { FeServices } from '../../FeServices.js';
+
+export class LoginPage extends LitElement {
+  static get styles() {
+    return css`
+      ${defaultStyles}
+
+      .hidden {
+        display: none;
+      }
+    `;
+  }
+
+  constructor() {
+    super();
+    this.isError = false;
+  }
+
+  async login(detail) {
+    this.isError = false;
+    try {
+      const data = await FeServices.postRequest({
+        url: '/login',
+        data: {
+          username: detail.username,
+          password: detail.password,
+        },
+      });
+
+      this.dispatchEvent(new CustomEvent('id', { detail: data.id }));
```

Comment:
Instead of generic id, use some specific name like customer-id or other something like that

Reply:
okay the id will be changed to customer-id